### PR TITLE
Main/develop

### PR DESCRIPTION
**Pull request by nishanthprakash**

### CODE REVIEW

**Comment by franzliedke** [comment]
This has a lot of strange indentation changes.

What exactly does the resizer change do?

**Comment by nishanthprakash** [comment]
This resizer change sets up the resizer's initial position, i.e. at the edge of the first video stream.

**Comment by jprberlin** [comment]
Please check indentation

**Comment by nishanthprakash** [comment]
My previous commit had a lot of bad indentation which I corrected with the 'resizer init' commit (this new 'resizer init' commit had an unintended indentation change which I reverted in my indentation commit) -- sorry about this. So the indentations seem in place now. Is there a particular indentation issue you want me to look into and revise?

File: video-player.html [hunk 1]
```diff
@@ -224,7 +223,9 @@
           this.stream = this.videoStreamList[0];
         }.bind(this));
 
-        // manage bindings
+        document.querySelector("#resizer").style.left = this.videoStreamList[0].style.width;
+
+	// manage bindings
```
Comment (franzliedke) [inline video-player.html]: This is off

File: video-player.html [hunk 2]
```diff
@@ -305,39 +306,36 @@
         var numVideos = this.videoStreamList.length;
         // --------------------------------------------
 
+	document.querySelector("#resizer").style.left = this.videoStreamList[0].style.width;
```
Comment (franzliedke) [inline video-player.html]: This is off.

File: video-player.html [hunk 3]
```diff
@@ -305,39 +306,36 @@
...
+
         if (width(this.videoStreamList[0]) < (this.startWidth + detail.dx)) {
           // We're making the first video bigger
-          if ((height(this.videoStreamList[0]) <= height(this.videoStreamList[1])) &&
-                  (document.documentElement.clientHeight <= computedHeight(this) + controlbarHeight ) ||
-                  (document.documentElement.clientHeight > computedHeight(this) + controlbarHeight )) {
+          if ((height(this.videoStreamList[0]) <= height(this.videoStreamList[1]))&&
+                   (document.documentElement.clientHeight <= computedHeight(this) + controlbarHeight ) ||
+                   (document.documentElement.clientHeight > computedHeight(this) + controlbarHeight )){
```
Comment (franzliedke) [inline video-player.html]: Please don't remove the spaces around the `&&` and before the opening `{`.

File: video-player.html [hunk 4]
```diff
@@ -305,39 +306,36 @@
...
                 this.videoStreamList[0].style.width = (this.startWidth + detail.dx).toString() + "px";
         	this.videoStreamList[1].style.width = (this.startWidth - detail.dx).toString() + "px"; 		
 	 }
           // Resize the first video, and then the second
           this.videoStreamList[0].resize(this.videoStreamList.length);
           this.videoStreamList.slice(-1)[0].resize(this.videoStreamList.length);
-	
-	this.videoStreamList[1].resize(this.videoStreamList.length);
+	  this.videoStreamList[1].resize(this.videoStreamList.length);
```
Comment (franzliedke) [inline video-player.html]: This is off.

File: video-player.html [hunk 5]
```diff
@@ -305,39 +306,36 @@
...
           this.videoStreamList.slice(-1)[1].resize(this.videoStreamList.length);
 
         } else if (width(this.videoStreamList[0]) > (this.startWidth + detail.dx)) {
           // We're making the first video smaller
-          if ((height(this.videoStreamList[0]) >= height(this.videoStreamList[1])) &&
-                  (document.documentElement.clientHeight <= computedHeight(this) + controlbarHeight ) ||
-                  (document.documentElement.clientHeight > computedHeight(this) + controlbarHeight )) {
+          if ((height(this.videoStreamList[0]) >= height(this.videoStreamList[1]))&&
+                   (document.documentElement.clientHeight <= computedHeight(this) + controlbarHeight ) ||
+                   (document.documentElement.clientHeight > computedHeight(this) + controlbarHeight )){
```
Comment (franzliedke) [inline video-player.html]: Same here.

File: video-player.html [hunk 6]
```diff
@@ -305,39 +306,36 @@
...
                 this.videoStreamList[0].style.width = (this.startWidth + detail.dx).toString() + "px";
 		this.videoStreamList[1].style.width = (this.startWidth - detail.dx).toString() + "px";
           }
           // Resize the second video, and then the first
           this.videoStreamList.slice(-1)[0].resize(this.videoStreamList.length);
           this.videoStreamList[0].resize(this.videoStreamList.length);
-       	this.videoStreamList.slice(-1)[1].resize(this.videoStreamList.length);
+	  this.videoStreamList.slice(-1)[1].resize(this.videoStreamList.length);
```
Comment (franzliedke) [inline video-player.html]: There's also a space missing here.

**Comment by franzliedke** [comment]
I commented on a few. ;)

Which editor are you using? If compatible, I would suggest you install an [EditorConfig plugin](http://editorconfig.org/), I will push an appropriate config file soon.

**Comment by nishanthprakash** [comment]
Thanks Franz. I use vi. The indentation seems appropriate as can be seen here: https://github.com/nishanthprakash/HTML5Player/blob/main/develop/video-player.html. 

I am not sure why the diffs are showing up such bad indentation.

**Comment by franzliedke** [comment]
Hmm, not sure what is going on, but I can definitely see these faulty changes in the [patch file for this pull request](https://patch-diff.githubusercontent.com/raw/openHPI/HTML5Player/pull/16.patch).

**Comment by nishanthprakash** [comment]
Maybe my definitions of "1 tab=8 spaces" differs from definition on other systems? I will just use spaces and see if this is resolved.

**Comment by nishanthprakash** [comment]
yup, tabs were the issue.

**Pull request merged**